rhessys/util/readtag_worldfile.c

```diff
@@ -89,8 +89,9 @@ param *readtag_worldfile(int *paramCnt, FILE *file,char *key){
 	    if(strcmp(strbuf2,"n_basestations")==0){
 	      break;
 	    }
+	    
```

Comment:
Thanks Brian! 
I totally agree with you that hash table should be much better.  I haven't create any hash table before and I need your help on this one. 

Thanks very much!